src/Makefile.am

```diff
@@ -43,6 +43,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/include        \
 
 bin_PROGRAMS = curl
 
+SUBDIRS = ../docs
```

Comment:
That became really weird. My response was regarding the docs/Makefile.am comment. I *think* the `../docs` approach should work...